agent/providers/lib/v8.go

```diff
@@ -108,7 +108,8 @@ func (v *V8) Deploy() error {
 		err = msg.RespondMsg(&nats.Msg{
 			Data: val,
 			Header: nats.Header{
-				agentapi.NexRuntimeNs: []string{strconv.FormatInt(runtimeNanos, 10)},
+				agentapi.NexRuntimeNs:    []string{strconv.FormatInt(runtimeNanos, 10)},
+				agentapi.OtelTraceparent: msg.Header.Values(agentapi.OtelTraceparent),
```

Comment:
never manually add a traceparent field to a nats header. Always use the current context and inject, which takes a trace context and does the header injection for you (keeping that operation opaque) 

https://pkg.go.dev/go.opentelemetry.io/otel/propagation#TraceContext.Inject